config.yaml

```diff
@@ -0,0 +1,82 @@
+#------------------------------------------------------------
+# Values for this lesson.
+#------------------------------------------------------------
+
+# Which carpentry is this (swc, dc, lc, or cp)?
+# swc: Software Carpentry
+# dc: Data Carpentry
+# lc: Library Carpentry
+# cp: Carpentries (to use for instructor training for instance)
+# incubator: The Carpentries Incubator
+carpentry: 'incubator'
+
+# Overall title for pages.
+title: 'Introduction to deep learning'
+
+# Date the lesson was created (YYYY-MM-DD, this is empty by default)
+created: '2023-05-26'
```

Comment:
The lesson was first created like 3 years ago right?

Reply:
Check I changed it to 2020-10-17